migrations.lock

```diff
@@ -211,6 +211,7 @@ blobs
  0014_alter_deletedblobmeta_id
 campaign
  0001_initial
+ 0002_dashboardmapreport
```

Comment:
unless not feasible, would be good merge these migrations.
I believe we generally do that on HQ to limit the number of migrations.